code/modules/games/cas.dm

```diff
@@ -0,0 +1,40 @@
+// CARDS AGAINST SPESS
+// This is a parody of Cards Against Humanity (https://en.wikipedia.org/wiki/Cards_Against_Humanity)
+// which is licensed under CC BY-NC-SA 2.0, the full text of which can be found at the following URL:
+// https://creativecommons.org/licenses/by-nc-sa/2.0/legalcode
+// Original code by Zuhayr, Polaris Station, ported with modifications
+
+/obj/item/weapon/deck/cas
+	name = "\improper CAS deck (white)"
+	desc = "A deck for the game Cards Against Spess, still popular after all these centuries. Warning: may include traces of broken fourth wall. This is the white deck."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "cas_deck_white"
+	var/card_face = "cas_white"
+	var/blanks = 10
+	var/list/card_text_list = list()
+
+/obj/item/weapon/deck/cas/black
+	name = "\improper CAS deck (black)"
+	desc = "A deck for the game Cards Against Spess, still popular after all these centuries. Warning: may include traces of broken fourth wall. This is the black deck."
+	icon_state = "cas_deck_black"
+	card_face = "cas_black"
+	blanks = 0
+
+/obj/item/weapon/deck/cas/New()
+	..()
+	if (card_face == "cas_white")
+		card_text_list = file2list("config/cas_white.txt")
```

Comment:
Indeed, instead of this sillyness just replace "card_face" var with the var that points to the location of the file and statically load it from the location stated in said var